src/base/bittorrent/session.cpp

```diff
@@ -274,6 +274,7 @@ Session::Session(QObject *parent)
     , m_IPFilterFile(BITTORRENT_SESSION_KEY("IPFilter"))
     , m_announceToAllTrackers(BITTORRENT_SESSION_KEY("AnnounceToAllTrackers"), false)
     , m_announceToAllTiers(BITTORRENT_SESSION_KEY("AnnounceToAllTiers"), true)
+    , m_asyncIOThreads(BITTORRENT_SESSION_KEY("AsyncIOThreads"), 4)
```

Comment:
Maybe "AsyncIOThreadsCount"?

Reply:
OK.

Reply:
Count is redundant here IMO. This isn't one of the possibly many methods/names in the code that should be as explicit as possible because it can be used for many different reasons.

Reply:
My suggestion is for config param name only, not for method name.

Reply:
Yes that's what i meant. In code you can have e.g. IOThreadsCount IOThreadsIterator etc... Here it can only mean count.

Reply:
I see no reason to use less descriptive names in configuration file. What's benefit from it?

Reply:
Because it's implied.. never mind.